<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistUI</class>
 <widget class="QWidget" name="PlaylistUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QWidget" name="_addRemoveFromPlaylistWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Maximum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>39</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="_title">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="_playlistViewsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>16777215</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="_mainWidget" native="true">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="_playlistStackedWidget">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string/>
        </property>
        <property name="currentIndex">
         <number>4</number>
        </property>
        <widget class="QWidget" name="_gridViewPage">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>280</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="_listViewPage">
         <property name="minimumSize">
          <size>
           <width>350</width>
           <height>200</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QWidget" name="_coverViewPage">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="_tvPage"/>
        <widget class="QWidget" name="_settingsPage"/>
        <widget class="QWidget" name="_newsPage"/>
        <widget class="QWidget" name="_aboutPage"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
